usecase/src/test/java/org/mobilitydata/gtfsvalidator/usecase/ParseSingleRowForFileTest.java

```diff
@@ -17,291 +17,136 @@
 package org.mobilitydata.gtfsvalidator.usecase;
 
 import org.junit.jupiter.api.Test;
-import org.mobilitydata.gtfsvalidator.domain.entity.ParsedEntity;
 import org.mobilitydata.gtfsvalidator.domain.entity.RawEntity;
 import org.mobilitydata.gtfsvalidator.domain.entity.RawFileInfo;
 import org.mobilitydata.gtfsvalidator.usecase.notice.base.ErrorNotice;
-import org.mobilitydata.gtfsvalidator.usecase.notice.base.InfoNotice;
-import org.mobilitydata.gtfsvalidator.usecase.notice.base.Notice;
-import org.mobilitydata.gtfsvalidator.usecase.notice.base.WarningNotice;
 import org.mobilitydata.gtfsvalidator.usecase.notice.error.CannotConstructDataProviderNotice;
 import org.mobilitydata.gtfsvalidator.usecase.port.GtfsSpecRepository;
 import org.mobilitydata.gtfsvalidator.usecase.port.RawFileRepository;
 import org.mobilitydata.gtfsvalidator.usecase.port.ValidationResultRepository;
+import org.mockito.*;
 
 import java.util.*;
 
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.core.IsInstanceOf.instanceOf;
-import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
 
 class ParseSingleRowForFileTest {
 
     @Test
     void shouldValidateAndParseOneByOne() {
 
-        MockResultRepo resultRepo = new MockResultRepo();
-        MockSpecRepo specRepo = new MockSpecRepo();
+        RawFileRepository.RawEntityProvider mockProvider = mock(RawFileRepository.RawEntityProvider.class);
+        when(mockProvider.hasNext()).thenReturn(true, true, true, false);
+        RawEntity testRawEntity = new RawEntity(Map.of("testKey","testValue"), 0);
+        when(mockProvider.getNext()).thenReturn(testRawEntity);
+
+        GtfsSpecRepository.RawEntityParser mockParser = mock(GtfsSpecRepository.RawEntityParser.class);
+        when(mockParser.validateNonStringTypes(any(RawEntity.class))).thenReturn(Collections.emptyList());
+        when(mockParser.parse(any(RawEntity.class))).thenReturn(null);
```

Comment:
This can be removed as it is the default behavior of a mockito mock

Reply:
Fixed